apps/Dataman/src/screens/branchSend.tsx

```diff
@@ -0,0 +1,78 @@
+import { ScrollView, View } from "react-native";
+import React from "react";
+import NavBar from "../components/NavBar";
+import { Button, List, Text } from "@ant-design/react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+const BranchSend = () => {
+  return (
+    <View>
+      <NavBar />
+      <ScrollView>
+        <List>
+          <List.Item>
+            <Text
+              style={{
+                fontSize: 28,
+                fontWeight: "600",
+                paddingVertical: 5,
+              }}
+            >
+              Branch to Branch Send
+            </Text>
+          </List.Item>
+          <List.Item arrow="horizontal">
```

Comment:
refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view